drivers/google/compute_util.go

```diff
@@ -194,13 +194,6 @@ func (c *ComputeUtil) createInstance(d *Driver) error {
 		return err
 	}
 
-	instance, err = c.instance()
```

Comment:
@jeffmendoza Sorry, didn't know that. Do you want me to do a PR about this or do you prefer to do it yourself?